src/ol/MapBrowserEventHandler.js

```diff
@@ -303,29 +324,36 @@ class MapBrowserEventHandler extends Target {
     // To avoid a 'false' touchmove event to be dispatched, we test if the pointer
     // moved a significant distance.
     if (this.isMoving_(pointerEvent)) {
+      this.updateActivePointers_(pointerEvent);
       this.dragging_ = true;
       const newEvent = new MapBrowserEvent(
         MapBrowserEventType.POINTERDRAG,
         this.map_,
         pointerEvent,
-        this.dragging_
+        this.dragging_,
+        undefined,
+        this.activePointers_
       );
       this.dispatchEvent(newEvent);
     }
   }
 
   /**
-   * Wrap and relay a pointer event.  Note that this requires that the type
-   * string for the MapBrowserEvent matches the PointerEvent type.
+   * Wrap and relay a pointermove event.
    * @param {PointerEvent} pointerEvent Pointer
    * event.
    * @private
    */
-  relayEvent_(pointerEvent) {
+  relayMoveEvent_(pointerEvent) {
     this.originalPointerMoveEvent_ = pointerEvent;
     const dragging = !!(this.down_ && this.isMoving_(pointerEvent));
     this.dispatchEvent(
-      new MapBrowserEvent(pointerEvent.type, this.map_, pointerEvent, dragging)
+      new MapBrowserEvent(
+        MapBrowserEventType.POINTERMOVE,
```

Comment:
For readability: renamed method from `relayEvent_` to `relayMoveEvent_`, and use a onstant event type, because this is only called for `pointermove`.